src/shared/backend/services/flashJobs.ts

```diff
@@ -42,11 +42,18 @@ export const startExecution = async (
   jobId: string,
   args: {
     device: USBDevice;
-    firmware: { data?: Buffer; url?: string; target: string };
+    firmware: {
+      data?: Buffer;
+      url?: string;
+      target: string;
+      version: string;
+      selectedFlags?: { name: string; value: string }[];
+    };
   },
-  { dfu, firmwareStore }: Context
+  { dfu, firmwareStore, cloudbuild }: Context
 ): Promise<void> => {
   let firmwareData = args.firmware.data;
+  const fetchCloudbuild = !!args.firmware.selectedFlags;
```

Comment:
Again, just `cloudBuild`

Reply:
seems a bit risky, there is also cloudbuid object with the methods..